packages/react-native-test-renderer/src/renderer/index.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import type {Element, ElementType} from 'react';
+
+import * as FabricUIManager from './FabricUIManagerMock';
+
+import {act} from 'react-test-renderer';
+import ReactFabric from 'react-native/Libraries/Renderer/shims/ReactFabric';
+
+type ReactNode = {
+  children: Array<ReactNode>,
+  props: {...},
+  viewName: string,
```

Comment:
Is this the host component name?

Reply:
What is the purpose of `instanceHandle`?
How `memoizedProps` compare to regular `props`?